tsconfig.json

```diff
@@ -3,8 +3,8 @@
     // tsc is insanely drunk with internal resolutions currently, this at least silences it so we can ship things
     // the installed versions are still the same as before, but the bumped tsc version might have made it stricter
     "skipLibCheck": true,
-    "target": "ES2016",
-    "module": "commonjs",
+    "target": "ESNext",
+    "module": "ESNext",
```

Comment:
we should test and alpha release to confirm everything still works as intended when importing in web if you haven't. i feel like there were build issues surrounding the need for commonjs at some point for reasons I can't recall, but that was also half a decade ago... just sanity check

Reply:
I did try installing the verdaccio build in web, which worked fine for MM... Though definitely worth publishing an alpha instead of a regular release here so we can tackle changes if needed 🙏🏽

Reply:
Full e2e paranoia test looking good:

- Native: https://jam.dev/c/9a441771-5018-4737-bc73-7dd3f0297432 (confirmed native unlock and wallet create is also happy).
  - noticing these warnings though seemingly unrelated? 
  <img width="605" alt="image" src="https://github.com/user-attachments/assets/5eb22c79-ad69-4f01-affe-b9d9aee48de9">
- MM w/ snap: https://jam.dev/c/bcb3d215-0df7-435a-8fff-d47e5e828091
- Ledger: https://jam.dev/c/fff60d54-7fdd-4b1b-8a42-7e723975caf4
  - Unrelated to this PR, but noticing the ack isn't happy for RUNE since the "Pending for review" screen rugs the device detection, and it's an effective throw in swapper